tests/src/borough_sausage_links/geometry.json

```diff
@@ -1005,12 +1005,12 @@
               51.501094561991984
             ],
             [
-              -0.09205544955937478,
```

Comment:
I used the web tool to visually diff. The examples I listed in the PR description explain the changes -- some roads that were imported very incorrectly are now removed. Some will likely return as footpath types soon